foxpuppet/foxpuppet.py

```diff
@@ -3,11 +3,11 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
-from .windows import WindowManager
+from .windows.window_manager import WindowManager
```

Comment:
I'm always up for imports from folders. For those which are most often used we can even include them into the top-level __init__.py.